charts/argocd-image-updater/templates/rbac.yaml

```diff
@@ -24,6 +24,15 @@ rules:
       - events
     verbs:
       - create
+  - apiGroups:
```

Comment:
📝 Ref: https://github.com/argoproj-labs/argocd-image-updater/blob/v0.15.1/manifests/base/rbac/argocd-image-updater-clusterrole.yaml

Reply:
No. We need to check if the definition is same as upstream and you didn't show it in this PR. So I put the ref.